nameko/amqp.py

```diff
@@ -38,12 +39,20 @@ class TestTransport(Transport):
     Connection = ConnectionTester
 
 
-def verify_amqp_uri(amqp_uri):
-    connection = Connection(amqp_uri)
+def verify_amqp_uri(amqp_uri, config=None):
```

Comment:
i know config is still up in the air, but eventually we might even move to something like a dict with all amqp config, e.g.

```
amqp:
  url: amqp://localhost,
  ssl: 
    ca_certs: ...
    ...
```

(or even)

```
amqp:
  default:
    url: amqp://localhost,
    ssl: 
      ca_certs: ...
      ...
  other:
    url: ...
```